payloads/Makefile

```diff
@@ -2,3 +2,4 @@ OREBOOT=$(abspath $(CURDIR)/../)
 include ../Makefile.inc
 ciclippy: clippy
 citest: test
+cicoverage: coverage
```

Comment:
please keep it simple and expand on the `citest` target instead (in all the `Makefile`s)

Reply:
Are you sure? I spent a lot of time to split the tasks. I believed it was better like that because now we can test with no coverage.

Reply:
We can already do this. That's what the simple `test` target is for. :)